.github/workflows/pr-check.yml

```diff
@@ -127,7 +127,11 @@ jobs:
         shell: bash
 
       - name: Run Tests
-        run: cargo test --frozen --all-features
+        run: cargo test --frozen --all-features -- --nocapture
+        shell: bash
+
+      - name: Find Environments
+        run: cargo run --release --target ${{ matrix.target }}
```

Comment:
This is harmless, given this runs in < 100ms
I want this running everytime so we can see the results.